### PR TITLE
Add section on further resources

### PR DESCRIPTION
**Pull request by marbre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ and for the Arm [Corstone-300](https://developer.arm.com/Processors/Corstone-300
 
 See the [Getting Started](docs/getting_started.md) guide for instructions on how to build this project.
 
+## Further Resources
+
+* [Running on a bare-metal platform](https://iree.dev/guides/deployment-configurations/bare-metal/); IREE.dev Guide.
+* [TinyIREE: An ML Execution Environment for Embedded Systems From Compilation to Deployment](https://doi.ieeecomputersociety.org/10.1109/MM.2022.3178068); H.-I. C. Liu, M. Brehler, M. Ravishankar, N. Vasilache, B. Vanik and S. Laurenzo; *IEEE Micro*, vol. 42, no. 5, pp. 9-16, Sept.-Oct. 2022.
+
```
Comment (lucas-camp) [inline README.md:28]: Is this blank line intentional?

Comment (marbre) [reply README.md:28]: It is :)

Comment (marbre) [reply README.md]: No it was not. Good catch, fixed.

**Review by lucas-camp — APPROVED** [review]
LGTM

**Pull request merged**